### DISCUSSION TITLE
Under what circumstances the GPU will occupy the video memory but the running gpu process pid is not displayed, but the log output of pytorch lightning tells me GPU available: True, used: True.

### DISCUSSION THREAD

**Discussion by qianjinhao** [Lightning-AI/pytorch-lightning#8283] [General]
Under what circumstances the GPU will occupy the video memory but the running gpu process pid is not displayed, but the log output of pytorch lightning tells me GPU available: True, used: True. I use gpus=[0] to specify the parameters of the trainer to confirm the gpu. The training speed is very slow, and it is estimated that GPU acceleration is not used. 
![image](https://user-images.githubusercontent.com/26296120/124441434-406f0e00-ddae-11eb-82c8-cf9888b9ffeb.png)

![image](https://user-images.githubusercontent.com/26296120/124441325-26353000-ddae-11eb-84e7-02b84a3c7ac2.png)

**Comment by awaelchli** [comment]
Try restarting the machine to release the memory. And maybe update to the lastest nvidia driver, i.e., 460.73.01
It could also simply be that this is the memory occupied by your desktop environment, if there is running a graphical one.
